gammapy/estimators/points/core.py

```diff
@@ -667,13 +667,30 @@ def plot(self, ax=None, sed_type=None, energy_power=0, time_format="iso", **kwar
         flux = scale_plot_flux(flux=flux.to_unit(flux_unit), energy_power=energy_power)
         if "time" in flux.geom.axes_names:
             flux.geom.axes["time"].time_format = time_format
-        ax = flux.plot(ax=ax, **kwargs)
-        ax.set_ylabel(f"{sed_type} [{ax.yaxis.units.to_string(UNIT_STRING_FORMAT)}]")
-        ax.set_yscale("log")
+            ax = flux.plot(ax=ax, **kwargs)
+        else:
+            ax = flux.plot(ax=ax, **kwargs)
+            ax.set_xlabel(f"Energy [{ax.xaxis.units.to_string(UNIT_STRING_FORMAT)}]")
+            ax.set_xscale("log", nonpositive="clip")
```

Comment:
why is the X axis formatting necessary here?

Reply:
Because I removed those 2 lines from the original function such that the new function only deal with the y-axis. This makes the new function usable for both FluxPoints and LigthCurve.